h/models/organization.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+
+import sqlalchemy as sa
+import slugify
+
+from h.db import Base
+from h.db import mixins
+
+
+ORGANIZATION_NAME_MIN_LENGTH = 4
```

Comment:
I know you grabbed this directly from `Group` 'cause I totally recognize it. Something to keep in mind, but likely not important at all for this PR, is that the 4-character lower limit in groups names has actually bitten us a couple times. Consider certain 3-character news organizations or whatnot...

This is easy enough to change in the future, though.

Reply:
Oooh...good point didn't think about that! I'll adjust the min length to 1 I think...It would be a little weird to be 1 char but if it's a acronym or nickname 1 letter might actually make sense.